src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/commands.py

```diff
@@ -0,0 +1,96 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+
+def load_command_table(self, _):
+    from azure.cli.core.commands import CliCommandType
+    from azure.cli.command_modules.eventhubs._client_factory import (namespaces_mgmt_client_factory,
+                                                                     event_hub_mgmt_client_factory,
+                                                                     consumer_groups_mgmt_client_factory,
+                                                                     disaster_recovery_mgmt_client_factory)
+    from .custom import empty_on_404
+
+    eh_namespace_util = CliCommandType(
+        operations_tmpl='azure.mgmt.eventhub.operations.namespaces_operations#NamespacesOperations.{}',
+        client_factory=namespaces_mgmt_client_factory
+    )
+
+    eh_event_hub_util = CliCommandType(
+        operations_tmpl='azure.mgmt.eventhub.operations.event_hubs_operations#EventHubsOperations.{}',
+        client_factory=event_hub_mgmt_client_factory
+    )
+
+    eh_consumer_groups_util = CliCommandType(
+        operations_tmpl='azure.mgmt.eventhub.operations.consumer_groups_operations#ConsumerGroupsOperations.{}',
+        client_factory=consumer_groups_mgmt_client_factory
+    )
+
+    eh_geodr_util = CliCommandType(
+        operations_tmpl='azure.mgmt.eventhub.operations.disaster_recovery_configs_operations#DisasterRecoveryConfigsOperations.{}',
+        client_factory=disaster_recovery_mgmt_client_factory
+    )
+
+# Namespace Region
+    custom_tmpl = 'azure.cli.command_modules.eventhubs.custom#{}'
+    eventhubs_custom = CliCommandType(operations_tmpl=custom_tmpl)
+    with self.command_group('eventhubs namespace', eh_namespace_util, client_factory=namespaces_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_namespace_create')
+        g.command('show', 'get', exception_handler=empty_on_404)
+        g.custom_command('list', 'cli_namespace_list', exception_handler=empty_on_404)
+        g.command('delete', 'delete')
+        g.command('exists', 'check_name_availability')
+        g.generic_update_command('update', custom_func_name='cli_namespace_update', custom_func_type=eventhubs_custom)
```

Comment:
Arguments with default values...

Reply:
the parameter help text don't have the default values.